### PR TITLE
Adding Managing policies docs

### PR DESCRIPTION
**Pull request by JoaoMartins51**

### CODE REVIEW

File: content/docs/details/managing-policies/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Policies Management
+publishedDate: '2024-11-05T21:00:00.0Z'
+description: How to configure and manage policies in Roadie for access control.
+---
+
+## Introduction
+
+In Backstage, **policies** are integral components of the permissions framework, governing access control within the platform. They define the rules and conditions under which users can perform specific actions on various resources, such as services, components, or documentation.
```
Comment (Xantier) [inline content/docs/details/managing-policies/index.md:9]:
```suggestion
In Roadie, **policies** are integral components of the permissions framework, governing access control within the platform. They define the rules and conditions under which users can perform specific actions on various resources, such as services, components, or documentation.
```

File: content/docs/details/managing-policies/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+**Key aspects of policies:**
+
+- **Authorization Decisions:** Policies determine whether a user is permitted to execute a particular action. For instance, a policy might allow only the owner of a service to delete it from the catalog. [Backstage](https://backstage.io/docs/permissions/writing-a-policy/)
+- **Conditional Logic:** Policies can incorporate complex conditions, such as verifying if a user belongs to a specific group or possesses certain attributes, to make nuanced authorization decisions. [Backstage](https://backstage.io/docs/permissions/custom-rules/)
+- **Customization:** Administrators have the flexibility to craft custom policies tailored to their organization's unique requirements, ensuring that access controls align with internal processes and security standards. [Backstage](https://backstage.io/docs/permissions/custom-rules/)
+
+By implementing policies, Backstage ensures that access to resources is managed effectively, enhancing security and maintaining the integrity of the development environment.
```
Comment (Xantier) [inline content/docs/details/managing-policies/index.md:17]:
```suggestion
By implementing policies, Roadie ensures that access to resources is managed effectively, enhancing security and maintaining the integrity of the development environment.
```

File: content/docs/details/managing-policies/index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: Policies Management
+publishedDate: '2024-11-05T21:00:00.0Z'
+description: How to configure and manage policies in Roadie for access control.
+---
+
+## Introduction
```
Comment (Xantier) [inline content/docs/details/managing-policies/index.md:7]: Change this to not be written from Backstage point of view but from Roadie POV and referring to and  on top of Backstage permissions/policies.

File: content/docs/details/managing-policies/index.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+In Roadie, **policies** are integral components of the permissions framework, governing access control within the platform. They define the rules and conditions under which users can perform specific actions on various resources, such as services, components, or documentation.
+
+**Key aspects of policies:**
+
+- **Authorization Decisions:** Policies determine whether a user is permitted to execute a particular action. For instance, a policy might allow only the owner of a service to delete it from the catalog. [Backstage](https://backstage.io/docs/permissions/writing-a-policy/)
```
Comment (Xantier) [inline content/docs/details/managing-policies/index.md:13]: The links might need a reference that they are going to Backstage docs and why

Comment (JoaoMartins51) [reply content/docs/details/managing-policies/index.md]: do you think my last chance covers that?

File: content/docs/details/permissions/index.md [hunk 5]
```diff
@@ -6,13 +6,15 @@ description: How to configure access control permissions in Roadie.
 
 ## Introduction
 
+**This feature is currently in the process of being rolled out. If you want access, please contact our support or sales teams.**
```
Comment (Xantier) [inline content/docs/details/permissions/index.md:9]: Is this true? I believe this is enabling to everyone

Comment (JoaoMartins51) [reply content/docs/details/permissions/index.md]: I changed it to the policies document, my mistake. Roles management has been rolled out, just not Policies management

**Review by Xantier — APPROVED** [review]

**Pull request merged**